Formula/ledit.rb

```diff
@@ -1,8 +1,9 @@
 class Ledit < Formula
   desc "Line editor for interactive commands"
   homepage "http://pauillac.inria.fr/~ddr/ledit/"
-  url "http://pauillac.inria.fr/~ddr/ledit/distrib/src/ledit-2.04.tgz"
-  sha256 "484897016b001b6d64eb24e38a213177b230ebf8e0d9808a969b62f012e70ee4"
+  url "https://github.com/chetmurthy/ledit/archive/ledit-2-05.tar.gz"
```

Comment:
Is this from a fork of `ledit`? The homepage still has the latest version at 2.04.

Reply:
And no one else seems to have switched to this fork, so I don't think using it is what we want to do:

https://repology.org/project/ledit/versions

Reply:
Yeah... but a kinda official fork I think (it's published don opam under [ledit.2.05](https://opam.ocaml.org/packages/ledit/)), the 2.05 only adds fixes for camlp5 compat.

Reply:
> And no one else seems to have switched to this fork, so I don't think using it is what we want to do:

Hm... I'd think official opam registry is the source of truth for OCaml packages, no?

Reply:
>Hm... I'd think official opam registry is the source of truth for OCaml packages, no?

Should it be? I genuinely don't know. Why does no one else seem to be aligning their version of `ledit` with opam's?

Reply:
> Should it be? I genuinely don't know.

From my point of view — yes. Also in this case the current maintainer of ledit on opam is Chet Murthy, he's also maintaining camlp5 as well (ledit 2.05 was released with fixed related to camlp5). Chet Murthy is also a visible member of OCaml community (I see them being a frequent poster/commenter on https://discuss.ocaml.org/).

Furthermore... I don't think the original author is maintaining ledit or camlp5 anymore, from the [OCaml discuss](https://discuss.ocaml.org/t/camlp5-and-ocaml-4-08/3985/14):

> Daniel (the author of camlp4 and camlp5, and the current maintainer of camlp5) was born and has lived most of his life in a pre-GitHub era. The fact that the project is on GitHub unfortunately does not mean that he is at ease with the way pull requests work. Furthermore, he is less interested in this project now (even if he continues using it for his personal OCaml projects, such as ledit). His current activities have more to do with formalizing math in Coq.


> Why does no one else seem to be aligning their version of ledit with opam's?

Good question.

Anyway, what other options do you think we have? We can vendor patches on top 2.04 in brew but it probably doesn't make much sense.

Reply:
Do we have any other sources that support Chet Murthy taking over the maintenance of `ledit`? The link you have there shows that Chet has taken over `camlp5`, but not `ledit`, unless I've missed something in the thread.

Reply:
Hi @chetmurthy (sorry for the ping) can we treat your fork on GitHub of the ledit repo as the one which is in active maintenance and is the official version of ledit going forward?

Reply:
Hello, yes, you can: Daniel and Pierre Boutillier both agreed that I should host a github repo, and will do whatever maintenance is needed.

Reply:
Great; sounds good. It would be nice if the new repo was linked at http://pauillac.inria.fr/~ddr/ledit/ so that other distros/package managers can see that they've got an outdated version of `ledit`.

Thanks, @chetmurthy.

Reply:
I'll get on that (changing the web page).